joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/passes/callgraph/DynamicCallLinker.scala

```diff
@@ -134,6 +139,47 @@ class DynamicCallLinker(cpg: Cpg) extends SimpleCpgPass(cpg) {
     }
   }
 
+  /** Attempts to refine possible call targets using points-to information.
+    */
+  private def filterWithVariableTypeInformation(
+    call: Call,
+    tgts: mutable.LinkedHashSet[String]
```

Comment:
I may be missing something, but it looks like `tgts` doesn't need to be mutable?